int16 status
int16 zed
int16 shoulder
int16 elbow
int16 yaw
int16 wrist1
int16 wrist2
int16 gripper
